build.gradle

```diff
@@ -53,14 +53,19 @@ dependencies {
 task createAllStartScripts() << {
      // just a placeholder
 }
-  def scripts = [ 'topicPublisher':'com.solace.samples.TopicPublisher',
+  def scripts = ['topicPublisher':'com.solace.samples.TopicPublisher',
 				 'topicSubscriber':'com.solace.samples.TopicSubscriber',
 				 'queueProducer':'com.solace.samples.QueueProducer',
 				 'queueConsumer':'com.solace.samples.QueueConsumer',
 				 'basicRequestor':'com.solace.samples.BasicRequestor',
 				 'basicReplier':'com.solace.samples.BasicReplier',
 				 'confirmedPublish':'com.solace.samples.ConfirmedPublish',
-				 'topicToQueueMapping':'com.solace.samples.TopicToQueueMapping'
+				 'topicToQueueMapping':'com.solace.samples.TopicToQueueMapping',
```

Comment:
Could we prefix these names with "feature"

Reply:
Done and pushed.